### PR TITLE
Chore/improve home page new project state

### PR DESCRIPTION
**Pull request by joshenlim**
Attached recording shows original in the first half, followed by updated

https://user-images.githubusercontent.com/19742402/191926495-1fec77ed-a9ae-428d-a884-81c978b41945.mov

### CODE REVIEW

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 1]
```diff
@@ -1,28 +1,246 @@
...
+                      }
+                    >
+                      SQL editor
+                    </Button>
+                  </a>
+                </Link>
+                <Link href="https://supabase.com/docs/guides/database">
+                  <a>
+                    <Button type="default" icon={<IconExternalLink size={14} />}>
+                      About Database
```
Comment (saltcod) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:83]: `About Database` feels not quite right.  `Database Docs` maybe?

Comment (saltcod) [reply studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:83]: aaah I see.   They're all `About <tool>` links

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 2]
```diff
@@ -1,28 +1,246 @@
...
+              </div>
+            </div>
+            <GetStartedHero />
+          </div>
+
+          <div className="flex flex-col justify-between h-full space-y-6">
+            <div className="space-y-2">
+              <h3 className="text-xl text-scale-1200">Explore our other products</h3>
+              <p className="text-base text-scale-1100">
+                Supabase provides all the backend features you need to build a product. You can use
```
Comment (saltcod) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:96]: This sentence feels a little wide — maybe a max-w-2xl on the container or have the second sentence on its own line? 

CleanShot 2022-09-23 at 09 55 35@2x

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 3]
```diff
@@ -1,28 +1,246 @@
...
+            <div className="space-y-2">
+              <h3 className="text-xl text-scale-1200">Welcome to your new project</h3>
+              <p className="text-base text-scale-1100">
+                Your project has been deployed on its own instance, with its own API all set up and
+                ready to use.
+              </p>
+            </div>
+          </div>
+
+          <div className="grid grid-cols-12 flex gap-4">
```
Comment (saltcod) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:31]: Has both flex and grid classnames

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 4]
```diff
@@ -1,28 +1,246 @@
...
+            <div className="col-span-7 space-y-8 flex flex-col justify-center">
+              <div className="space-y-2">
+                <h3 className="text-xl text-scale-1200">
+                  Get started by building out your database
```
Comment (saltcod) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:35]: Feels like something might be needed between these sections?

CleanShot 2022-09-23 at 10 00 30@2x


An <hr> maybe? Not sure
CleanShot 2022-09-23 at 10 01 21@2x

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 5]
```diff
@@ -1,28 +1,246 @@
...
+          <div className="space-y-2">
+            <h3 className="text-xl text-scale-1200">Connecting to your new project</h3>
+            <p className="lg:max-w-sm text-scale-1100">
+              Interact with your database through the{' '}
+              <Link href="https://supabase.com/docs/reference">
+                <a className="text-brand-900">Supabase client libraries</a>
+              </Link>{' '}
+              with your API keys.
+            </p>
+            <p className="lg:max-w-sm text-scale-1100">
```
Comment (saltcod) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:218]: maybe a text-sm for this second paragraph? 

CleanShot 2022-09-23 at 10 04 37@2x

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx [hunk 6]
```diff
@@ -1,28 +1,246 @@
...
-    </div>
-    <div className="col-span-4">
-      <div className="space-y-2">
-        <h3 className="text-xl text-scale-1200">Connecting to your new project</h3>
+const NewProjectPanel: FC<Props> = ({}) => {
+  const router = useRouter()
+  const { ref } = router.query
+
+  return (
+    <div className="grid grid-cols-12 gap-20">
```
Comment (saltcod) [inline studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:18]: I forget: do we want an lg:grid on these so they collapse on mobile, or do we want the horizontal scroll?

**Review by saltcod — APPROVED** [review]

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx (hunk 4 above)
Comment (joshenlim) [reply studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:35]: think this should be okay - was using stripe's docs as one of my references https://stripe.com/docs

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx (hunk 6 above)
Comment (joshenlim) [reply studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:18]: ahh yeap - i'll make them all into a column for mobile

File: studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx (hunk 5 above)
Comment (joshenlim) [reply studio/components/interfaces/Home/NewProjectPanel/NewProjectPanel.tsx:218]: hmm i'll leave this as the base font size for now for consistency across the text elements on this page specifically

**Pull request merged**